packages/expo-modules-autolinking/src/reactNativeConfig/__tests__/reactNativeConfig-test.ts

```diff
@@ -160,6 +162,42 @@ describe(findDependencyRootsAsync, () => {
   });
 });
 
+describe(resolveAppProjectConfigAsync, () => {
+  it('should return app project config for android', async () => {
```

Comment:
```suggestion
  it('should return app project config for Android', async () => {
```

Reply:
haha i'm leaning toward to keep test case names as lowercases 😅